compiler/rustc_error_codes/src/error_codes/E0547.md

```diff
@@ -0,0 +1,27 @@
+#![stable(since = "1.0.0", feature = "test")]
```

Comment:
I feel like a big part of the file "disappeared". :rofl:

Reply:
This was strange xD